src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php

```diff
@@ -654,6 +662,14 @@ public function theOrderShouldHavePaymentState(OrderInterface $order, $orderPaym
     }
 
     /**
+     * @Then the order :order should have order shipping state :orderPaymentState
```

Comment:
:orderShippingState

Reply:
Also below the parameter variable should be `$orderShippingState`